addons/base_import/models/base_import.py

```diff
@@ -937,6 +946,16 @@ def parse_preview(self, options, count=10):
                         matches[index] = match.split('/')
             elif options.get('has_headers'):
                 matches = self._get_mapping_suggestions(headers, header_types, fields_tree)
+                if self._context.get("lang"):
```

Comment:
Use self.env.context, not much longer but more standard :)

Reply:
Changed